### ISSUE TITLE
Battery upgrade output?

### ISSUE THREAD

**Issue by dvdvdvlol** [P3pp3rF1y/SophisticatedBackpacks#1245]
I put this here cus its not a bug but maybe. I use create new age(adds electricity etc..) and with the electrical connector if I put it on the backpack it can charge it but there is no way to decharge it except manually with the slots so basically it can charge but cannot charge things 
![image](https://github.com/user-attachments/assets/7fd9c9d7-5d7d-479e-aabb-c93b7e3b2b36)

**Comment by P3pp3rF1y** [comment]
The reason I am pretty sure is that you actually need something that would actively pull the charge from the backpack because it doesn't proactively push the energy out of itself. I want to add a way (probably in an advanced upgrade) of defining which directions it pulls from / pushes to so I will leave this open for that enhancement.

**Comment by dvdvdvlol** [comment]
Thanks for the answer and even if I have something like a motor or battery in need of electricity it doesnt send electricity, and I dont see any other ways to "pull electricity"?

**Comment by P3pp3rF1y** [comment]
It can't send it so you would have to have something that pulls it, like for example I know that mekanism energy pipes can be set to pull mode and that works, but machines that just expect some energy to be pushed to them wouldn't.

**Comment by dvdvdvlol** [comment]
Yeah it is not a lot of mods that do that tho, thanks for the consideration ;)
